openedx/features/badging/views.py

```diff
@@ -18,7 +23,7 @@ def trophycase(request):
 
     # Get course id and course name of courses user is enrolled in
     enrolled_courses_data = CourseEnrollment.enrollments_for_user(user).order_by(
-        'course__display_name').values_list('course_id', 'course__display_name')
+        'course__display_name').values_list(COURSE_ID_KEY, 'course__display_name')
```

Comment:
can we make it consistent.. As for one param we are using constant and for one we are using string.

Reply:
Done